spec/paperclip/storage/s3_spec.rb

```diff
@@ -644,47 +700,81 @@ def counter
     end
   end
 
+  # for aws-sdk-v2 the bucket.name is determined by the :s3_region
   context "Parsing S3 credentials with a s3_host_name in them" do
     before do
-      rebuild_model storage: :s3,
-        bucket: 'testing',
-        s3_credentials: {
-          production: { s3_host_name: "s3-world-end.amazonaws.com" },
-          development: { s3_host_name: "s3-ap-northeast-1.amazonaws.com" }
-        }
-        @dummy = Dummy.new
+      if defined?(::Aws)
+        rebuild_model storage: :s3,
+          bucket: 'testing',
+          s3_credentials: {
+          production: {
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.